test/list.test.ts

```diff
@@ -22,7 +22,7 @@ describe("commands/list", function () {
     );
   });
 
-  test("throws without chain", async function () {
+  test("List throws without chain", async function () {
```

Comment:
you don't like my nested naming structure? 😜

Reply:
the test run should print things like:
commands/list
   throws without chain

so you get the "list" context from the describe above. anyway, if it makes your life easier, keep the prefixes!

Reply:
Yeah the logs from the test appear between the description name and the test name, making it a little tougher to figure out which "throws without chain" test you're looking at sometimes.